### ISSUE TITLE
Testing configuration file

### ISSUE THREAD

**Issue by pazur** [circus-tent/circus#819]
There should be some tool (preferably just an option for circusd) to verify configuration file.

The simplest way to do this is running circus.config.get_config(). But it would be nice if this feature validated also if used sockets and environmental variables are declared.

**Comment by holstvoogd** [comment]
+1

**Comment by Eagllus** [comment]
+1 yes please

**Comment by aswen** [comment]
Yeah this would be really useful when circus is used in an environment where the services it manages should be available at all times.
